modules/ROOT/pages/dataweave-cookbook-java-methods.adoc

```diff
@@ -210,6 +210,73 @@ import valueOf from java!java::lang::String
 valueOf(true)
 ----
 
+== Java Bridge Example
+
+The following example shows an API that returns a greeting message based on the input `uriParam`.
+The Java bridge calls a static method within the `my.class.org.Greeter` class that builds the response message.
+
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/json
+import java!my::class::org::Greeter
+---
+{
+response: Greeter::greet(attributes.uriParams."name"
+}
+----
+
+=== XML for Java Bridge Example
+
+Paste this code into your Studio XML editor to quickly load the flow for this example into your Mule app:
```

Comment:
Technically speaking, the English language should always take a doer (subject) of an action (verb).
Adding a "You should" or rephrasing the sentence will do as well:
To quickly load the flow for this example into your Mule app, you should paste this code into your Studio XML editor:

Reply:
Thank you. Though, I'll leave the sentence as it is because it's part of a template that was already approved by our previous editor. 
Also, we have to avoid using "should" per our writing guidelines:
"These verbs have multiple meanings, creating ambiguity, and they make content hard to localize because many languages have no equivalent to modal verbs."
Refer to the "Modal Verbs" section in https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference